pkg/virt-controller/watch/topology/tsc.go

```diff
@@ -163,3 +161,14 @@ func vmiHasInvTSCFeature(vmi *k6tv1.VirtualMachineInstance) bool {
 	}
 	return false
 }
+
+func isVmiUsingHyperVReenlightenment(vmi *v1.VirtualMachineInstance) bool {
```

Comment:
I believe the build error comes from here

```suggestion
func isVmiUsingHyperVReenlightenment(vmi *k6tv1.VirtualMachineInstance) bool {
```